### ISSUE TITLE
Please fix or revert the flight ringkydink

### ISSUE THREAD

**Issue by Entoarox** [MattDahEpic/RingkyDinks#9]
The old rinkydink behaviour meant that sometimes you failed to get flying properly.

Now it just randomly cuts out and drops you in/on whatever was below, like say.. lava lakes in the nether...

Just please, dear god please, FIX THAT! >_<

**Comment by MattDahEpic** [comment]
I've done some work on this in beta2, as far as I can tell this is mainly based on the server tickrate. I've added a possible fix in beta2, let me know if it works.

**Comment by Entoarox** [comment]
Unable to fly with beta 2 at all, not even when in creative!

**Comment by MattDahEpic** [comment]
Try enabling the dink first, it should be glowing.

**Comment by Entoarox** [comment]
Even with item consumption turned off?
Edit: And the no flying even in creative, I meant no flying AT ALL, not even when you're without a dink

**Comment by MattDahEpic** [comment]
Yes, it's meant for turning off the functionality of dinks when not needed. (You probably don't want the lava walking dink on when harvesting lava.)

**Comment by MattDahEpic** [comment]
Please test in alpha5 and report if fixed!

**Comment by Entoarox** [comment]
The flight ring doesnt work at all in alpha 5, not with fuel or without, activated or deactivated, as bauble or not.
